DirectProgramming/DPC++FPGA/Tutorials/Features/dsp_control/README.md

```diff
@@ -0,0 +1,186 @@
+
+| Optimized for                     | Description
+---                                 |---
+| OS                                | Linux* Ubuntu* 18.04/20.04, RHEL*/CentOS* 8, SUSE* 15; Windows* 10
+| Hardware                          | Intel® Programmable Acceleration Card (PAC) with Intel Arria® 10 GX FPGA <br> Intel® FPGA Programmable Acceleration Card (PAC) D5005 (with Intel Stratix® 10 SX) <br> Intel® FPGA 3rd party / custom platforms with oneAPI support <br> *__Note__: Intel® FPGA PAC hardware is only compatible with Ubuntu 18.04* 
+| Software                          | Intel® oneAPI DPC++ Compiler <br> Intel® FPGA Add-On for oneAPI Base Toolkit 
+| What you will learn               |  How to apply global DSP control in command-line interface. <br> How to apply local DSP control in source code. <br> Scope of datatypes and math operations that support DSP control.
+| Time to complete                  | 15 minutes
+
+## Purpose
+This tutorial shows how to apply global and local DSP controls to set implementation preference for math operations to control usage of DSPs. The global control is applied via command-line option and affects math operations in all kernels. The local control is applied via library function and affects math operations in a block scope in one single kernel. Both global and local controls only affect math operations that support DSP control.
+
+### Scope of Datatypes and math operations
+| Datatypes              | math operations
+---                      |---
+| `float`                | Add, sub, constant mul
+| `hls_float<8, 23>`     | Add, sub, constant mul
+| `int`                  | Constant mul
+| `ac_int`               | Constant mul
+| `ac_fixed`             | Constant mul
+
+**NOTE:** _constant mul_ means one operand of the multiplication is a constant.
+
+### Global Control
+The `-Xsdsp-mode` command-line option sets implementation preference of math operations that support DSP control in all kernels. It has three valid options:
+| Option                           | Explanation
+---                                |---
+| `-Xsdsp-mode=default`            | This is the default. The compiler determines the implementation based on datatype and math operation.
+| `-Xsdsp-mode=prefer-dsp`         | Prefer math operations to be implemented in **DSPs**. If a math operation is implemented by DSPs by default, you will see no difference in resource utilization or area. Otherwise, you will notice decrease in the usage of soft-logic resources and increase in the usage of DSPs.
+| `-Xsdsp-mode=prefer-softlogic`   | Prefer math operations to be implemented in **soft-logic**. If a math operation is implemented without DSPs by default, you will see no difference in resource utilization or area. Otherwise, you will notice decrease in the usage of DSPs and increase in the suage of soft-logic resources.
```

Comment:
Otherwise, you will notice *a* decrease in the usage of DSPs and *an* increase in the ~~suage~~ usage of soft-logic resources.